ibis_tpc/h14.py

```diff
@@ -14,7 +14,7 @@ def tpc_h14(con, DATE="1995-09-01"):
     q = q.filter([q.l_shipdate >= DATE, q.l_shipdate < add_date(DATE, dm=1)])
 
     revenue = q.l_extendedprice * (1 - q.l_discount)
-    promo_revenue = q.p_type.like("promo%").ifelse(revenue, 0)
```

Comment:
is there an `ilike()` or another way to do case-insensitive LIKE?

Reply:
Yep [`ilike`](https://ibis-project.org/api/expressions/strings/#ibis.expr.types.strings.StringValue.ilike) is a thing!

Reply:
I guess the question is how closely we want to hew to the TPC-H queries as they are defined